### PR TITLE
chore(root,components): add AGPL 3 license

### LINKED ISSUE #161: Add license
This repository doesn't have a license yet. @chaoran-chen which one should we add?

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #161

### PR Checklist

- ~~[ ] The implemented feature is covered by an appropriate test.~~

### CODE REVIEW

File: components/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "version": "0.0.1",
     "description": "GenSpectrum web components for building dashboards",
     "type": "module",
+    "license": "AGPL-3.0-or-later",
```
Comment (fengelniederhammer) [inline components/package.json:7]: See https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license

Comment (chaoran-chen) [reply components/package.json:7]: Where do you see the "or-later"? I can't find it on the page.

Comment (fengelniederhammer) [reply components/package.json:7]: https://spdx.org/licenses/ shows it. We could also use `-only`. I don't really know what the difference is.

Comment (chaoran-chen) [reply components/package.json:7]: Ah, interesting. Let's better fix it to `-only`.

Comment (fengelniederhammer) [reply components/package.json]: Done

**Pull request merged**